app/editor/src/store/slices/workorder/interfaces/IWorkOrderModel.ts

```diff
@@ -4,7 +4,7 @@ import {
   IUserModel,
   WorkOrderStatusName,
   WorkOrderTypeName,
-} from '..';
+} from 'hooks';
 
 export interface IWorkOrderModel extends IAuditColumnsModel {
```

Comment:
This is an interface that represents a model from the API.   Why are you moving it here?